### ISSUE TITLE
need process bool in right way

### ISSUE THREAD

**Issue by webclerk** [quandyfactory/dicttoxml#67]
```
def test_xml_result():
        dict_data = {
            'token': None,
            'version': None,
            'create_time': 12345,
            'succeed': True,
            'messages': [],
            'data': {'A': 1}
        }
        print(dicttoxml(dict_data))
```
the result is
```
<?xml version="1.0" encoding="UTF-8" ?><root><token type="null"></token><version type="null"></version><create_time type="int">12345</create_time><succeed type="bool">True</succeed><messages type="list"></messages><data type="dict"><A type="int">1</A></data></root>
```
In fact, True is not the xml bool value type, it should be true.
We fixed this issue, please check the attachment file, it's the code.

[dicttoxml_fix_bool_issue.txt](https://github.com/quandyfactory/dicttoxml/files/3025283/dicttoxml_fix_bool_issue.txt)

**Comment by svobora** [comment]
You library generates invalid xml files due to True and False as boolean values, instead of correct true and false. Why don't you fix this capitalization problem? It is a one-liner.

**Comment by mangin** [comment]
https://github.com/quandyfactory/dicttoxml/pull/84

**Comment by quandyfactory** [comment]
This issue is fixed in version 1.7.8.
